src/app.rs

```diff
@@ -236,7 +237,7 @@ impl<AppData: Send + Sync + 'static> App<AppData> {
 ///
 /// This type is useful only in conjunction with the [`HttpService`] trait,
 /// i.e. for hosting a Tide app within some custom HTTP server.
-#[derive(Clone)]
```

Comment:
[question]
Is there any problem with deleting Clone here?

Reply:
Oh no. This is a mistake! Thanks for the catch!

Reply:
Fixed! Thanks.